test_manager/features/environment.py

```diff
@@ -62,16 +63,41 @@ def before_all(context):
     counter = 1
     all_languages = []
     all_transports = []
+    all_uris = []
     while context.config.userdata.get(f"uE{str(counter)}") is not None:
         all_languages.append(context.config.userdata[f"uE{str(counter)}"])
         all_transports.append(context.config.userdata[f"transport{str(counter)}"])
         context.logger.info(all_languages)
+
+        current_uri = {
+            "data": {
+                "uuri": {
+                    "ue_id": random.randrange(0, 0x7FFF),
+                    "ue_version_major": 1,
+                    "resource_id": random.randrange(0, 0x7FFF),
+                }
+            }
+        }
+
+        while current_uri in all_uris:
+            current_uri = {
+                "data": {
+                    "uuri": {
+                        "ue_id": random.randrange(0, 0x7FFF),
+                        "ue_version_major": 1,
+                        "resource_id": random.randrange(0, 0x7FFF),
+                    }
+                }
+            }
+
+        all_uris.append(current_uri)
```

Comment:
C++ L1 check for default uri includes 

`	if (uuri.resource_id() != 0) {
		return {false, Reason::BAD_RESOURCE_ID};
	}
`